src/predict/api/helpers/utils.py

```diff
@@ -205,6 +202,33 @@ def get_gp_predictions(airqloud):
     records = list(db.gp_predictions.find(query, projection))
     return records
 
+def get_gp_predictions(airqloud):
+    '''
+    returns pm 2.5 predictions for a particular airqloud
+    '''
+    # try:
+    #     client = MongoClient(MONGO_URI)
+    # except pymongo.errors.ConnectionFailure as e:
+    #     return {'message':'unable to connect to database', 'success':False}, 400
+
+    # db = client[DB_NAME]
+    if airqloud == None:
+        records = get_all_gp_predictions()
+    else:
+        query = {'airqloud':airqloud}
+        projection = {'_id': 0, 
+                        'latitude': 1, 
+                        'longitude': 1, 
+                        'predicted_value': 1, 
+                        'variance': 1, 
+                        'interval': 1, 
+                        'airqloud':1, 
+                        'created_at':1, 
+                        'airqloud_id':1, 
+                        'values':1}
```

Comment:
@uman95 Thanks for the update.

How will we deal with airqlouds that don't have the **values** attribute?

Reply:
@FaithDaka **don't have values attribute** mean **no prediction**. I don't think there's a decision on that yet.
![image](https://user-images.githubusercontent.com/32173551/192386712-4144f1e5-5ff9-4cdf-8e1a-4110ff23ed2d.png)